package.json

```diff
@@ -100,6 +100,7 @@
   "dependencies": {
     "@escape.tech/mookme": "2.4.1",
     "@redux-devtools/remote": "0.8.0",
+    "@web3modal/ethers": "4.1.0",
```

Comment:
🙈

Reply:
yeah wow that's insane. it comes from web3 modal too and not https://github.com/ethers-io/ethers.js D: